mep/books/management/commands/reconcile_oclc.py

```diff
@@ -26,7 +31,23 @@ class Command(BaseCommand):
         # db notes last
         'Notes']
 
+    #: summary message string for each mode
+    summary_message = {
+        'report': 'Processed %(count)d items, found matches for %(found)d',
+        'update': 'Processed %(count)d items, updated %(updated)d',
+    }
+
+    progbar = None
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.stats = defaultdict(int)
+
+        self.script_user = User.objects.get(username=settings.SCRIPT_USERNAME)
+        self.item_content_type = ContentType.objects.get_for_model(Item).pk
+
     def add_arguments(self, parser):
+        parser.add_argument('mode', choices=['report', 'update'])
```

Comment:
would it make sense for this to have a default?

Reply:
I don't think shouldn't update by default; it seems a little weird to me to report by default, although I guess not terrible? I don't expect we'll run this very often - once to get as many things as possible reconciled and then occasionally to reconcile records that have been cleaned up by the team.